### PR TITLE
Update getting started with flow tutorial

### PR DESCRIPTION
**Pull request by shadikhani**

### CODE REVIEW

File: tutorials/getting-started-with-flow/01__setting-up-the-project/content.adoc [hunk 1]
```diff
@@ -80,30 +80,58 @@ Let's take a look at some of the generated files:
...
+** The `MainView` constructor:
 
-<2> Tells Vaadin to activate automatic https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps[PWA] features. This annotation is optional.
+*** Creates a text field to enter user's name.
 
-<3> Makes this view a `VerticalLayout` which shows components vertically.
+*** Creates a button with the text *Say hello* on it to send a greeting message.
 
-<4> Creates a button with the text `Click me` on it.
+*** GreetService that has a method and it checks if the user has filled the text field or is an anonymous user.
```
[the remark this answers is not in the document]
Comment (shadikhani) [reply tutorials/getting-started-with-flow/01__setting-up-the-project/content.adoc]: This part has been removed

File: tutorials/getting-started-with-flow/01__setting-up-the-project/content.adoc [hunk 2]
```diff
@@ -80,30 +80,58 @@ Let's take a look at some of the generated files:
...
+** The `@PWA` annotation tells Vaadin to activate automatic https://vaadin.com/pwa[PWA^] features. This annotation is optional.
+
+** The `@CssImport` Imports a CSS file that are mentioned in the paranthesis.
+
+** The view extends `VerticalLayout` which shows components vertically.
+
+** The `MainView` constructor:
 
-<2> Tells Vaadin to activate automatic https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps[PWA] features. This annotation is optional.
+*** Creates a text field to enter user's name.
```
[the remark this answers is not in the document]
Comment (shadikhani) [reply tutorials/getting-started-with-flow/01__setting-up-the-project/content.adoc]: Done

File: tutorials/getting-started-with-flow/01__setting-up-the-project/content.adoc [hunk 3]
```diff
@@ -80,30 +80,58 @@ Let's take a look at some of the generated files:
...
+        add(textField, button);
     }
 }
 ----
-<1> Tells Vaadin to direct the root URL to this view. The URL parameter is optional and will be derived from the class name if not given.
+** The `@Route` annotation tells Vaadin to direct the root URL to this view. The URL parameter is optional and is derived from the class name, if not given.
+
+** The `@PWA` annotation tells Vaadin to activate automatic https://vaadin.com/pwa[PWA^] features. This annotation is optional.
+
+** The `@CssImport` Imports a CSS file that are mentioned in the paranthesis.
```
[the remark this answers is not in the document]
Comment (shadikhani) [reply tutorials/getting-started-with-flow/01__setting-up-the-project/content.adoc]: Done

File: tutorials/getting-started-with-flow/01__setting-up-the-project/content.adoc [hunk 4]
```diff
@@ -80,30 +80,58 @@ Let's take a look at some of the generated files:
 [source,java]
 ----
 /**
- * The main view contains a button and a click listener.
+ * The main view contains a text field for getting the user name and a button
```
[the remark this answers is not in the document]
Comment (shadikhani) [reply tutorials/getting-started-with-flow/01__setting-up-the-project/content.adoc]: Done

File: tutorials/getting-started-with-flow/01__setting-up-the-project/content.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (alejandro-du) [reply tutorials/getting-started-with-flow/01__setting-up-the-project/content.adoc]: I think you mean "Imports the specified CSS file"

[the remark this answers is not in the document]
Comment (shadikhani) [reply tutorials/getting-started-with-flow/01__setting-up-the-project/content.adoc]: Done

**Pull request merged**